decidim-core/app/views/decidim/notification_mailer/event_received.html.erb

```diff
@@ -9,13 +9,30 @@
 <% end %>
 
 <% if @event_instance.try(:safe_resource_text).present? %>
+  <% if @event_instance.perform_translation? %>
+    <p style="font-weight: bold"><%= t(".original_text") %></p>
+  <% end %>
+
   <blockquote>
     <p>
       <%= @event_instance.safe_resource_text %>
     </p>
   </blockquote>
 <% end %>
 
+<% if @event_instance.content_in_same_language? %>
+  <p><%= t(".same_language", language: I18n.locale.to_s ) %></p>
+<% elsif @event_instance.translation_missing? %>
+  <p><%= t(".no_translation_available", link: @event_instance.resource_url ) %></p>
+<% elsif @event_instance.perform_translation? %>
+  <p style="font-weight: bold"><%= t(".translated_text") %></p>
```

Comment:
To be confirmed with @carolromero: should we show a text saying something like "this is an automated translation (etc etc)" text in the email?

Reply:
@andreslucena yes, this was already agreed upon.

Reply:
The .translated_text, has been changed from "Translated text" to "Automatically translated text:'